src/components/dashboard/main-content/course-enrollments/data/constants.js

```diff
@@ -1,22 +1 @@
-export const FETCH_COURSE_ENROLLMENTS_REQUEST = 'FETCH_COURSE_ENROLLMENTS_REQUEST';
-export const FETCH_COURSE_ENROLLMENTS_SUCCESS = 'FETCH_COURSE_ENROLLMENTS_SUCCESS';
-export const FETCH_COURSE_ENROLLMENTS_FAILURE = 'FETCH_COURSE_ENROLLMENTS_FAILURE';
-export const CLEAR_COURSE_ENROLLMENTS = 'CLEAR_COURSE_ENROLLMENTS';
-export const UPDATE_COURSE_RUN_STATUS = 'UPDATE_COURSE_RUN_STATUS';
-export const UPDATE_IS_MARK_COURSE_COMPLETE_SUCCESS = 'UPDATE_IS_MARK_COURSE_COMPLETE_SUCCESS';
-export const UPDATE_IS_UNARCHIVE_COURSE_SUCCESS = 'UPDATE_IS_UNARCHIVE_COURSE_SUCCESS';
```

Comment:
[sanity check] I'm assuming these constants are getting deleted as they're no longer used throughout the repo?

I know `COURSE_STATUSES` is removed in favor of defining it elsewhere, but just verifying these other constants.

Reply:
Yep. Grayed out on my IDE and highlighted as unused.

![Screenshot 2025-01-09 at 9 14 57 AM](https://github.com/user-attachments/assets/483cdfc0-a435-4f02-969e-e6eb7fe95de9)

Reply:
Sounds good. These were probably leftover from when we used Redux to manage this state, which makes sense if they're now unused.